### ISSUE TITLE
VNET/Private Endpoint

### ISSUE THREAD

**Issue by cedoula** [MicrosoftTranslator/DocumentTranslation#89]
Hello guys,

Does the Document Translation tool support TranslatorText resources configured with a VNET and/or private endpoint?

Thanks.
Cedric

**Comment by chriswendt1** [comment]
I am not sure. All URLs the app uses are fully qualifiable in the settings, so theoretically it should work, as long as you enter those URLs correctly.
If you think you have everything correct, try adding
`/translator/text/v3.0`
to the text translation endpoint. 

What have you tried and what are you observing?
What do you see when you hit 'Test' in the Settings page?

**Comment by cedoula** [comment]
Adding /translator/text/v3.0 fixes it!
![image](https://user-images.githubusercontent.com/68669675/168909163-40c19e89-b03e-49c5-a2a4-e69fb4bbf0f8.png)

Thanks!

**Comment by cedoula** [comment]
This could be added to the documentation for users using Vnet and PE.

**Comment by chriswendt1** [comment]
Super. Thanks for confirming this works.

**Comment by chriswendt1** [comment]
Added to the documentation:
https://microsofttranslator.github.io/DocumentTranslation/#virtual-networkprivate-endpoint

I would be thankful if you could review for accuracy, and whether this would have helped you in the first place.

**Comment by cedoula** [comment]
Hi Chris, thank you for the documentation update!
I have noticed another thing: when I activate VNET on the storage account now, the test passes but when I try to do a document translation the process gets stuck, see below.
![image](https://user-images.githubusercontent.com/68669675/169416461-58e533b8-ad91-45ac-b764-b3f56bea1821.png)

When I check the storage account in the portal, the document is indeed uploaded and the translator metrics shows successful calls, but the DT tool stays locked in the stage above.
I tried Postman, it worked and created the translated file. 
Have you observed this issue before?
